crypto/txscript/src/lib.rs

```diff
@@ -225,6 +285,7 @@ impl<'a, T: VerifiableTransaction, Reused: SigHashReusedValues> TxScriptEngine<'
             cond_stack: Default::default(),
             num_ops: 0,
             kip10_enabled,
+            runtime_sig_op_counting: None, // todo?
```

Comment:
I don't think it's needed, but explain why in a comment

Reply:
6ba349b096f149002f4eb4ba2afaeb479b68273c